client/py/examples/control/abort/nominal_sequence.py

```diff
@@ -0,0 +1,86 @@
+#  Copyright 2024 Synnax Labs, Inc.
+#
+#  Use of this software is governed by the Business Source License included in the file
+#  licenses/BSL.txt.
+#
+#  As of the Change Date specified in that file, in accordance with the Business Source
+#  License, use of this software will be governed by the Apache License, Version 2.0,
+#  included in the file licenses/APL.txt.
+
+import synnax as sy
+import time
+
+# We've logged in via the CLI, so there's no need to provide credentials here. See
+# https://docs.synnaxlabs.com/reference/python-client/get-started for more information.
+client = sy.Synnax()
+
+# Define the control channel names
+PRESS_VALVE = "press_vlv_cmd"
+VENT_VALVE = "vent_vlv_cmd"
+PRESSURE = "pressure"
+
+# Open a control sequence under a context manager, so that the control is released when
+# the block exits
+with client.control.acquire(
+    name="Press Sequence",
+    # Defines the authorities at which the sequence controls the valve channels.
+    #
+    # ####
+    # Notice that we take a higher control authority here than we do at the start of the
+    # abort sequence. This means that the abort sequence will take control OVER
```

Comment:
Could say specifically abort sequence is at authority 100 at the beginning so they see that in the other file